core/src/main/java/tc/oc/pgm/PGMPlugin.java

```diff
@@ -394,6 +384,17 @@ private void registerListeners() {
     registerEvents(new JoinLeaveAnnouncer(matchManager));
   }
 
+  private boolean loadInitialMaps() {
+    try {
+      mapLibrary.loadNewMaps(false).get(300, TimeUnit.SECONDS);
```

Comment:
Possible oversight: 5 minutes or 30 seconds?

```suggestion
      mapLibrary.loadNewMaps(false).get(30, TimeUnit.SECONDS);
```